### ISSUE TITLE
Encode per-unit indicator as true/false instead of 1/0

### ISSUE THREAD

**Issue by tasseff** [lanl-ansi/GasModels.jl#221]

**Comment by rb004f** [comment]
This will make GasModels consistent with the conventions of PowerModels

**Comment by tasseff** [comment]
Additionally, for greater consistency, the field should be called `per_unit` instead of `is_per_unit`.

**Comment by ccoffrin** [comment]
The key `per_unit` is a standard https://lanl-ansi.github.io/InfrastructureModels.jl/stable/developer/ it should not be changed unless done in other packages as well.

**Comment by rb004f** [comment]
Correct, this issue is to make GasModels consistent with the other packages.  Rename is_per_unit to per_unit and make it a boolean.
